numba/tests/test_array_methods.py

```diff
@@ -383,6 +383,17 @@ def check_err(arr, dtype):
                 run(arr, dtype)
             self.assertEqual(str(raises.exception),
                              "new type not compatible with array")
+        def check_err_noncontig_last_axis(arr, dtype):
+            # check NumPy interpreted version raises
```

Comment:
On first glance this seems a tiny bit odd to be checking NumPy, but I think the purpose that this serves is to explicitly test / document our own understanding of the NumPy behaviour we replicate in Numba. So I'm happy with this check staying in.

Out of curiousity, do we expect further NumPy changes around this area that we might detect in future releases with the NumPy check?

Reply:
I am unsure if there will be changes in NumPy in this area in future releases, but put this in catch NumPy either changing this or the associated error message(s).